lib/tasks/plugins.rake

```diff
@@ -88,3 +88,17 @@ namespace :redmine do
     end
   end
 end
+
+namespace :plugins do
+desc 'Adds Meeting plugin for now'
```

Comment:
Are the descriptions still valid?

I'll have to do further code lookup, but I hope that this will do more than add/remove the *Meeting* plugin.